static/game/hero.js

```diff
@@ -70,7 +73,7 @@ export class Hero extends Creature {
       this.invincibilityFrames > 0
         ? `rgba(255, 255, 255, ${this.invincibilityFrames /
             this.invincibilityFramesMax})`
-        : "black",
+        : "rgba(0, 0, 0, 0)",
       4,
       "white"
```

Comment:
Since enemies have a "DrawColor", shouldn't the hero as well? I can see eventualities where the hero's color should be edited.

Reply:
There's a few points where "white" are referenced in the file too, meaning it should probably be able to be changed in one place.
Alternatively, changing the eye color separately from the body color seems like it would be cool.